generate_print_file.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import csv
+import os
+from datetime import datetime
+from pathlib import Path
+from sqlalchemy import create_engine
+from sqlalchemy.sql import text
+from google.cloud import storage
+
+
+PRINT_FILE_TEMPLATE = ('UAC', 'QUESTIONNAIRE_ID', 'WALES_UAC', 'WALES_QUESTIONNAIRE_ID', 'ADDRESS_LINE1',
+                       'ADDRESS_LINE2', 'ADDRESS_LINE3', 'TOWN_NAME', 'POSTCODE', 'PRODUCTPACK_CODE')
+
+
+class QidQuantityMismatchException(Exception):
+    pass
+
+
+def _get_uac_qid_links(engine, questionnaire_type):
+    uac_qid_links_query = text("SELECT * FROM casev2.uac_qid_link WHERE SUBSTRING(qid from 1 for 2)"
+                               "= :questionnaire_type and caze_case_ref is null")
+
+    return engine.execute(uac_qid_links_query, questionnaire_type=questionnaire_type)
+
+
+def initialise_db():
+    db_port=os.getenv('db_port', "NoValue")
+    db_name=os.getenv('db_name', 'NoValue')
+    db_host=os.getenv('db_host', 'NoValue')
+    db_password_file = open("/root/.db-credentials/password", "r")
```

Comment:
It would make the script a lot easier to test locally if these were read from environment variables rather than files.